samples/bootloader/src/main.c

```diff
@@ -69,9 +69,11 @@ static void validate_and_boot(const struct fw_info *fw_info, uint16_t slot)
 
 	printk("Firmware version %d\r\n", fw_info->version);
 
+#ifdef CONFIG_SB_MONOTONIC_COUNTER
```

Comment:
We should use `if (IS_ENABLED(CONFIG_FOO))` when possible, can you use that here?